_rules/SC1-2-video-audio-description.md

```diff
@@ -125,10 +112,6 @@ There are no major accessibility support issues known for this rule.
 
 ## Inapplicable
 
```

Comment:
Include an inapplicable test case for streaming media

Reply:
We couldn't find a good way to do this. Please leave a suggestion if you have one.